include/cuco/detail/static_multiset/static_multiset_ref.inl

```diff
@@ -267,5 +267,68 @@ class operator_impl<
   }
 };
 
+template <typename Key,
+          cuda::thread_scope Scope,
+          typename KeyEqual,
+          typename ProbingScheme,
+          typename StorageRef,
+          typename... Operators>
+class operator_impl<
+  op::find_tag,
+  static_multiset_ref<Key, Scope, KeyEqual, ProbingScheme, StorageRef, Operators...>> {
```

Comment:
The `find` operator should be the same for all of the data structures, right? Can we unify the implementation somehow to reduce code duplication?